grep.json

```diff
@@ -0,0 +1,231 @@
+{ 
+    "#": [
+        "JSON doesn't allow for multi-line strings, so all outputs are presented ",
+        "here as arrays of strings. It's up to the test generator to join the ", 
+        "lines together with line breaks.",
+        "",
+        "The tests are divided into two groups: ",
+        "* Grepping a single file",
+        "* Grepping multiple files at once",
+        "",
+        "The language track implementing this exercise should",
+        "ensure that when the tests run, three files are created",
+        "with the following contents:",
+        "",
+        " [File #1]",
```

Comment:
what would you think of putting this in own section, like

```json
"files": [
  {
    "name": "iliad.txt",
    "contents": [
      "Achilles sing, O Goddess! Peleus' son;",
      "(other lines omitted from this comment)"
    ]
  }
]
```

I know it's not meant to be executed as a test, but perhaps it would make generating the files easier.